easybuild/easyconfigs/p/Pandoc/Pandoc-2.13.eb

```diff
@@ -0,0 +1,30 @@
+# This file is an EasyBuild reciPY as per https://easybuilders.github.io/easybuild/
+# Author: Pablo Escobar Lopez
+# sciCORE - University of Basel
+# SIB Swiss Institute of Bioinformatics 
+# Contribution from the NIHR Biomedical Research Centre
+# Guy's and St Thomas' NHS Foundation Trust and King's College London
+# uploaded by J. Sassmannshausen
+
+easyblock = 'Tarball'
+
+name = 'Pandoc'
+version = '2.13'
+
+homepage = 'https://pandoc.org'
+description = "If you need to convert files from one markup format into another, pandoc is your swiss-army knife"
+
+toolchain = SYSTEM
+
+source_urls = ['https://github.com/jgm/pandoc/releases/download/%(version)s/']
+sources = ['%(namelower)s-%(version)s-linux-amd64.tar.gz']
```

Comment:
Hmm, we should probably use a `versionsuffix` here, since they (now?) provide `arm64` builds too.

Reply:
If so, wouldn't it be better to provide multiple sources here instead?

Reply:
@smoors See https://github.com/smoors/easybuild-easyconfigs/pull/5

Reply:
There's a better option, we can avoid the `versionsuffix` for Pandoc, and stick to a single easyconfig file that supports both `x86_64` and `aarch64`, but we need https://github.com/easybuilders/easybuild-framework/pull/3670 for that...

Reply:
Huh, I thought we could do that already? Is it (currently) just implemented as a template variable for a few selected easyblocks (like Java) before?

Reply:
I'm also a bit confused.
a lot of easyconfigs already use `%(arch)s` in their sources string, why can't we use that here too?

Reply:
is it because we cannot translate `%(arch)s` into `amd64` or `arm64` in the easyconfig?

Reply:
Hm, I thought we even supported using a dictionary in sources, but maybe that's just dependencies (like Java) or checksums (like CUDA) does?